src/transformers/models/layoutlm/modeling_layoutlm.py

```diff
@@ -638,8 +638,8 @@ def _init_weights(self, module):
 
             `What are input IDs? <../glossary.html#input-ids>`__
         bbox (:obj:`torch.LongTensor` of shape :obj:`({0}, 4)`, `optional`):
-            Bounding Boxes of each input sequence tokens. Selected in the range ``[0, config.max_2d_position_embeddings
-            - 1]``.
```

Comment:
This is the only real failure of the upgraded script on the whole lib.
With the upgrade, the script now thinks that line `   - 1` is the start of a new list. I think this is an acceptable price to pay as there is an easy fix for those cases (just remove the space between the - and the previous word).

Reply:
could it lookeahed and see if there are more items? If there none, then it is not a list.

Reply:
The problem is that this logic is a bit complex to implement since the next item may be several lines ahead (since one item can take several lines). It's definitely possible, but I would like to only add it if we really need it since it will add some extra complexity (and thus execution time).

Reply:
It probably depends on how it's implemented. 

If it's a single paragraph we are looking at then it can be parsed directly into an optional header and a bullet list, then you immediately know whether there are multiple bullets or just one. so a hypothetical parser could go:
```
header, items = parse_to_bullet_list(para)
if len(items) < 2: this is not a bullet list.
```

Of course, this could still fail if the input happened to be:
```
Bounding Boxes of each input sequence tokens. Selected in the range ``[0, config.max_2d_position_embeddings
            - 1]``. blah blahof each input sequence tokens. Selected in the range ``[0, something_else
            - 5]``. 
```
So there is no winning it seems.